packages/api-electron/package.json

```diff
@@ -2,14 +2,15 @@
   "name": "containerjs-api-electron",
   "version": "0.0.2",
   "description": "",
-  "main": "index.js",
+  "main": "build/lib/preload.js",
```

Comment:
Hmm ... I'm not so sure this is correct. This package is being executed from the electron command line app, which uses the `main` property here to determine the application entry point:

https://electron.atom.io/docs/tutorial/quick-start/#write-your-first-electron-app